src/main/kotlin/org/rust/lang/core/psi/ext/RsLoopOnlyExpr.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.psi.ext
+
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.RsExpr
+
+// Expr that can only be used inside a `loop` or `while` block. E.g. `break`, `continue`
+interface RsLoopOnlyExpr : RsExpr, RsLabelReferenceOwner, RsOuterAttributeOwner {
```

Comment:
Name and comment are confusing because `break` can be used inside blocks with `#![feature(label_break_value)]` feature